StarterProject/package.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "webdriverio",
+  "description": "A Node.js bindings implementation for the W3C WebDriver protocol",
+  "version": "4.12.0",
+  "homepage": "http://webdriver.io",
+  "author": "Camilo Tapia <camilo.tapia@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/webdriverio/webdriverio.git"
+  },
+  "bugs": {
+    "url": "https://github.com/webdriverio/webdriverio/issues"
+  },
+  "license": "MIT",
+  "main": "./build/index.js",
+  "engines": {
+    "node": ">= 4.8.5"
+  },
+  "eslintConfig": {
+        "env": {
+            "browser": true
+        }
+    },
+  "scripts": {
+    "build": "run-s clean compile copy",
+    "clean": "run-p clean:*",
+    "clean:build": "rm -fr build",
+    "clean:coverage": "rm -fr coverage",
+    "compile": "run-p compile:*",
+    "compile:index": "mkdir -p build && babel index.js --out-file build/index.js",
+    "compile:lib": "babel lib/ -d build/lib",
+    "copy": "run-p copy:*",
+    "copy:packagejson": "cp package.json build",
+    "copy:ejstemplate": "cp lib/helpers/wdio.conf.ejs build/lib/helpers/wdio.conf.ejs",
+    "eslint": "eslint ./lib test/",
+    "release": "np patch",
+    "release:patch": "np patch",
+    "release:minor": "np minor",
+    "release:major": "np major",
+    "test": "run-s eslint test:unit",
+    "test:ci": "run-s eslint build test:$_ENV",
+    "test:desktop": "mocha test/setup.js test/spec/*.js test/spec/desktop/*.js",
+    "test:functional": "mocha test/setup.js test/spec/functional/**/*.js",
+    "test:multibrowser": "mocha test/setup.js test/spec/multibrowser/**/*.js",
+    "test:wdio": "mocha test/setup-unit.js test/spec/wdio/*.js",
+    "postversion": "run-s build",
+    "prepare": "npm prune",
+    "watch": "npm run compile:lib -- --watch",
+    "tidy": "prettier-eslint \"**/*.js\""
```

Comment:
I think we can wipe out a lot of these scripts that aren't really relevant.  Also I think you're missing a script that will run scripts with the wdio testrunner

Reply:
Yes, I have wiped them all out, updates soon!!